psi4/src/read_options.cc

```diff
@@ -1428,6 +1428,11 @@ int read_options(const std::string &name, Options & options, bool suppress_print
 
     /*- SUBSECTION DFT -*/
 
+    /*- The DFT combined functional name, e.g. B3LYP, or GEN to use a python reference to a
+        custom functional specified by DFT_CUSTOM_FUNCTIONAL. -*/
+    options.add_str("DFT_FUNCTIONAL", "HF");
+    /*- A custom DFT functional object (built by Python or NULL/None) -*/
+    options.add("DFT_CUSTOM_FUNCTIONAL", new PythonDataType());
```

Comment:
It looks like something went wrong in the merge, the above two options have been removed.

Reply:
Now I have fixed it. @dgasmith

Reply:
Looks like this is still here?